tests/triangles-test.js

```diff
@@ -0,0 +1,26 @@
+assert = require('chai').assert;
```

Comment:
Be sure to always use `var` or `let/const` when defining variables. I would use `const` here if you are using ES6